package.json

```diff
@@ -20,7 +20,10 @@
   "author": "Gnosis",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.17.1"
+    "@gnosis.pm/dex-contracts": "file:.yalc/@gnosis.pm/dex-contracts",
+    "bignumber.js": "^9.0.0",
```

Comment:
I don't see where this dependency is used.

Reply:
good point. It was part of the tutorial I was following, but looks like it's not needed.